CMSgov/eAPD
# dmirano/4332-cypress-with-LD-flags

Resolves #4332

### Description
Adds a command that lets Cypress mock Launch Darkly flag states. The flag data is stored within `launch-darkly-flags.json`.

### Significant changes or possible side effects
Added a Cypress command to each beforeEach() of tests that required validation to be on.  

### Automated test cases written
None, just added to existing tests

### Steps to manually verify this change

1. Verify validation LD flag is off for preview builds 
2. Verify Cypress tests in circle all pass

### This pull request is ready to code review when

      tests, refactor
      screenreader, text scaling) OR an exemption is documented

### This pull request is ready to test when


### This pull request is ready to review when the QA has


### This pull request can be merged when
